content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -379,6 +379,77 @@ are affected by the same inconsistencies as the regular restart policy. Kubelet
 runtime garbage collection, intermitted connectivity issues with the control plane may cause the state
 loss and containers may be re-run even when you expect a container not to be restarted.
 
+#### Restart All Containers {#restart-all-containers}
+
+{{< feature-state feature_gate_name="RestartAllContainersOnContainerExits" state="alpha" >}}
+
+If your cluster has the feature gate `RestartAllContainersOnContainerExits` enabled, you can specify
+`RestartAllContainers` as an action in `restartPolicyRules`. When a container's exit matches a rule with this action, the entire Pod is terminated and restarted in-place.
```

Comment:
```suggestion
`RestartAllContainers` as an action in `restartPolicyRules`. When a container's exit matches a rule with this action, the entire Pod is restarted in-place.
```

let's not say "terminated" as it may confuse things